docs/content/deployments.md

```diff
@@ -449,6 +449,18 @@ Note that the metaschemas [http://json-schema.org/draft-04/schema](http://json-s
 
 Similarly, the `allow_net` capability restricts what hosts the `http.send` built-in function may send requests to, and what hosts the `net.lookup_ip_addr` built-in function may resolve IP addresses for.
 
+### Features
```

Comment:
Additional information is added to this section in https://github.com/open-policy-agent/opa/pull/6346.